### ISSUE TITLE
Dropzone flickering when dropping files

### ISSUE THREAD

**Issue by rodrigoyoshida** [adopted-ember-addons/ember-file-upload#201]
The dropzone area flickers when I am dropping files, like it was being disabled and enabled again very quickly, causing the browser to open the local image instead of uploading it if I drop the file in the exact moment it was disabled. It happens on my app and on the demo app. I created a GIF to demonstrate the issue. 

![flicker2](https://user-images.githubusercontent.com/3301597/49873689-a7e59c80-fe03-11e8-8b85-ddf5530e324f.gif)

**Comment by ttill** [comment]
I also experience this using Chromium and Chrome (both (73.0.x). Does not appear in Firefox (66.0.3) though.

**Comment by ttill** [comment]
Both in Firefox and Chromium `dragleave` is fired when dragging over child-elements of the dropzone (`dragenter` is fired immediately afterwards again). Looks like the bug is somehow linked to this, because applying `pointer-events: none;` to all dropzone children, fixes the bug.

Strange thing is that the flickering does not always occur when dragging above a child-element. I was not able to figure out a movement that consistently produces this behaviour.

**Comment by beanshearer** [comment]
I have also experienced this but applying `pointer-events: none;` to all of the dropdown children did not fix this

**Comment by jelhan** [comment]
A reproduction as an Ember Twiddle would be helpful. Does someone has the time to set one up?

@beanshearer I had a quick look in your commit. Do I get it right that `dragenter` and `dragleave` are also called for child elements and bubble to this one? If so, did you considered having a check in `dragleave` event handler to ignore events fired by child elements? Something like:

```
if (event.target !== this.element) {
  // ignore events fired for child elements
  return;
}
````

**Comment by bmaehr** [comment]
@ttill Thanks for the hint!
`pointer-events: none;` by css on all childs of the `{{#file-dropzone ... }}` fixed it. 

Additionally make sure the div `<div class='dropzone' id={{dropzone.id}}>` is not getting smaller while dropping, otherwise you will get also filckering.

**Comment by gilest** [comment]
Should be fixed in `5.0.0-beta.8`. Please reopen if not
